hashicorp/boundary-ui
# refactor: 💡 Updated Manage Dropdown for hosts form

:tickets: ✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14602

# Description
Continuing refactoring upgrades for HDS manage dropdown menus.

## Screenshots (if appropriate)
**Before**
<img width="2032" alt="Screenshot 2024-08-02 at 10 30 36 AM" src="https://github.com/user-attachments/assets/69cac276-3e19-4bf3-b02d-12f9a5474035">
<img width="2032" alt="Screenshot 2024-08-02 at 10 30 41 AM" src="https://github.com/user-attachments/assets/99e27d2e-18e0-434c-a3e8-91685d07466a">
**After**
<img width="2032" alt="Screenshot 2024-08-02 at 10 34 09 AM" src="https://github.com/user-attachments/assets/6eae17d1-94e2-48e2-a299-83c48932a5db">
<img width="2032" alt="Screenshot 2024-08-02 at 10 34 14 AM" src="https://github.com/user-attachments/assets/0059e76d-78c6-4cd6-919f-0a59143d7b46">

## How to Test
1. Start up Boundary with yarn
2. Select a project from the org scope.
3. Select "Host Catalogs" from the sidebar
4. Select resource from table
5. Select "Hosts" and click on a resource in the table
6. HDS Changes should be reflected, only having a "Delete" attribute

## Checklist


~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [] I have added tests that prove my fix is effective or that my feature works~
